test/interpreter/operator-plus.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('Operator PLUS', () => {
     const engine = HyperFormula.buildFromArray([
       ['="2"+"3"'],
       ['="foobar"+1'],
-      ['\'3'],
+      ['3'],
       ['=A3+A3'],
```

Comment:
`=3+3`? This test is about "number coerce". And it should not change, the coercion with `'3` should still work correctly.

Reply:
There is no coercion with `'3`, it's just a string.

Reply:
Excel and GS does the coercion, this breaks the compatibility with both